packages/runtime/src/account.ts

```diff
@@ -137,6 +156,13 @@ export class StoreAccount implements StoreAccountI {
       throw new Error('Maximum created applications for an account is 10');
     }
 
+    // raise minimum balance
+    // https://developer.algorand.org/docs/features/asc1/stateful/#minimum-balance-requirement-for-a-smart-contract
+    this.minBalance += (
+      ALGORAND_ACCOUNT_MIN_BALANCE +
```

Comment:
why do we need it here ? i think we already have the account with `ALGORAND_ACCOUNT_MIN_BALANCE`

Reply:
This is different. `1 ALGO` is the 'base fee' for creating or opting-into app. You can read the link provided above

Reply:
Please refer to link, I think it's 10^5.

Reply:
Ah, yes. Thanks for catching. Will update